nvflare/fuel/hci/client/cli.py

```diff
@@ -237,7 +237,7 @@ def do_help(self, arg):
                 self.write_string("Server Commands")
                 self.write_string("---------------")
                 for cmd_name in server_cmds:
-                    self._show_one_command(cmd_name, self.api.server_cmd_reg)
+                    self._show_one_command(cmd_name, self.api.server_cmd_reg, show_invisible=True)
```

Comment:
why do we want to show invisible commands here?

Reply:
My question here should be, will we somehow show invisible commands even if the command name not 100% match?

Reply:
This is used when user types "help cmd", and the cmd help info is shown only if cmd name matches.
Invisible only means to hide the command from the command list shown to the user. The command is still active, and the user can use it if he knows the name.